profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.install

```diff
@@ -37,27 +39,37 @@ function nexteuropa_varnish_schema() {
 
 /**
  * Implements hook_requirements().
+ *
+ * At installation and run time phase, All Varnish feature settings' parameters
+ * must be set.
+ * The control status returns which parameters are missing.
  */
 function nexteuropa_varnish_requirements($phase) {
   $requirements = array();
   $t = get_t();
 
-  if ($phase == 'install') {
-    global $conf;
+  $control_phase = array(
+    'install',
+    'runtime',
+  );
 
+  if (in_array($phase, $control_phase)) {
     $required_vars = array(
-      'nexteuropa_varnish_http_targets',
-      'nexteuropa_varnish_request_method',
-      'nexteuropa_varnish_request_user',
-      'nexteuropa_varnish_request_password',
-      'nexteuropa_varnish_tag',
-      'nexteuropa_varnish_http_timeout',
+      'varnish_http_targets',
+      'varnish_request_method',
+      'varnish_request_user',
+      'varnish_request_password',
+      'varnish_tag',
+      'varnish_http_timeout',
     );
+    $settings = _nexteuropa_varnish_get_varnish_settings(FALSE);
 
     foreach ($required_vars as $var) {
-      if (!isset($conf[$var])) {
+      if (!isset($settings[$var])) {
+        $t_var = array('@var' => $var);
         $requirements[$var] = array(
-          'description' => $t('Following variable must be set: @var.', array('@var' => $var)),
+          'title' => $t('NextEuropa Varnish: "nexteuropa_@var" is missing', $t_var),
+          'description' => $t('Following variable must be set: nexteuropa_@var. Please ask your support team to check the server configuration.', $t_var),
```

Comment:
It is also displayed when you activate the related feature set. So, It is not so useless.